js/map_utils.js

```diff
@@ -265,17 +265,36 @@ function updateLegendText(d, category) {
   }, toolTipDelay);
 }
 
-function clearLegendText() {
+function updateLegendTextToView() {
 
-  waterUseViz.elements
-    .buttonBox
-    .selectAll("#legend-title")
-    .text("Water Use");
-
-  waterUseViz.elements
-    .buttonBox
-    .selectAll('.category-amount')
-    .text("");
+  if(activeView === 'USA') {
+    console.log('activeView is USA, so setting category amounts');
+    
+    waterUseViz.elements
+      .buttonBox
+      .selectAll("#legend-title")
+      .text("U.S. Water Use");
+  
+    d3.json("data/wu_data_15_sum.json", 
```

Comment:
we really should load and cache this data somewhere rather than loading it every time. but let's get it working first, then deal with that.